examples/camera-app/camera-common/include/camera-avstream-controller/camera-avstream-controller.h

```diff
@@ -0,0 +1,106 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.h>
+#include <app/util/config.h>
+#include <vector>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace CameraAvStreamManagement {
+
+/**
+ * The application interface to define the options & implement commands.
+ */
+class CameraAVStreamController : public CameraAVStreamMgmtDelegate
```

Comment:
I think instead of renaming the Management classes as Controllers, we could create a Controller interface class for the Management classes that require them and have the Management class implement the Controller interface alongwith the Delegate interface.
Manager seems more apt for an Impl class that implements both the delegate and the controller interface.
That way, also, the classes that do not have controller functions don't need any change at the moment.

Reply:
Acked